deepmd/dpmodel/fitting/property_fitting.py

```diff
@@ -131,6 +141,9 @@ def serialize(self) -> dict:
             **InvarFitting.serialize(self),
             "type": "property",
             "task_dim": self.task_dim,
+            "intensive": self.intensive,
+            "property_name": self.property_name,
+            "property_dim": self.property_dim,
```

Comment:
it is a breaking change. you need to update the version compatibility.

Reply:
Done